libs/ui/src/lib/progress-bar/progress-bar.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="flex bg-white rounded-t-lg rounded-b-lg">
```

Comment:
I would put the white outcircle in the component, i think it's the responsability of the container.
The component should just display the progress bar.